src/chapters/eda-part2/eda-part2.rst

```diff
@@ -1,26 +1,34 @@
 EDA With Python Part 2
 ======================
 
-Readings
---------
-
 Read the following articles, follow along where instructed:
 
-| `20 Must-Know Pandas Function for Exploratory Data Analysis <https://www.analyticsvidhya.com/blog/2021/04/20-must-known-pandas-function-for-exploratory-data-analysis-eda/>`__.
+Pandas Function
+---------------
+
+1. `20 Must-Know Pandas Function for Exploratory Data Analysis <https://www.analyticsvidhya.com/blog/2021/04/20-must-known-pandas-function-for-exploratory-data-analysis-eda/>`__.
 
 * Key takeaway: Reference list of common Pandas methods for EDA (some are review from last week).
 
-| `A Quick Guide to the Different Types of Outliers <https://www.anodot.com/blog/quick-guide-different-types-outliers/>`__.
+Outlier Detection
+-----------------
+
+2. `A Quick Guide to the Different Types of Outliers <https://www.anodot.com/blog/quick-guide-different-types-outliers/>`__.
 
```

Comment:
Under each new section ordered lists should restart and not continue from the previous section.

Use #. in front of link insted of hard coding.